int/configuration/configuration.go

```diff
@@ -0,0 +1,38 @@
+package configuration
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"runtime"
+)
+
+//nolint:lll
+const errMsgPath = "configuration path detection is not supported, use the MASSA_HOME environment variable to specify its location"
+
+// Path returns the default configuration directory for the current OS.
+// It checks the "MASSA_HOME" environment variable first. If "MASSA_HOME" is not set, it will
+// use OS-specific default paths.
+func Path() (string, error) {
+	// If the MASSA_HOME environment variable is set, use that
+	if massaHome := os.Getenv("MASSA_HOME"); massaHome != "" {
+		return massaHome, nil
+	}
+
+	switch {
+	case runtime.GOOS == "windows":
+		ex, err := os.Executable()
+		if err != nil {
+			return "", fmt.Errorf("getting executable path: %w", err)
+		}
+
+		return filepath.Dir(ex), nil
+
+	case runtime.GOOS == "linux", runtime.GOOS == "darwin":
```

Comment:
this function was previously `getConfigDir` declined in files `dirs_unix.go` and `dirs_windows.go`. I understand that folder `pkg` wasn't the right place for this function, but why are you assembling this function with `case runtime.GOOS`?

Reply:
because it easier to read that to navigate between 3 files as everything enter in one screen page

Reply:
true